### PR TITLE
Add support for Python 3.13

### PR DESCRIPTION
**Pull request by rokm**
A glimpse into the horrors that near-future holds...

### CODE REVIEW

**Comment by rokm** [comment]
I guess with 3.13.0rc1 being out (and supposedly being available on GHA), we can start considering merging this.

File: setup.cfg [hunk 1]
```diff
@@ -57,7 +57,7 @@ classifiers =
 packages = find:
 zip_safe = False
 include_package_data = False
-python_requires = >=3.8, <3.13
+python_requires = >=3.8, <3.14
```
Comment (bwoodsend) [inline setup.cfg:61]: Don't forget the `Programming Language :: Python :: 3.12` classifier as well

**Review by bwoodsend — APPROVED** [review]

**Comment by bwoodsend** [comment]
> I guess with 3.13.0rc1 being out (and supposedly being available on GHA), we can start considering merging this.

Yeah, I was thinking the same. Things should be settled enough in Python land by now.

File: setup.cfg (hunk 1 above)
Comment (rokm) [reply setup.cfg:61]: > Don't forget the `Programming Language :: Python :: 3.12` classifier as well

Good catch!

**Pull request merged**